ko/lessons/advanced/behaviours.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: page
+title: 비헤이비어
+category: advanced
+order: 10
+lang: ko
+---
+
+이전 강좌에서 타입스팩에 관해 배웠습니다. 여기서는그 사양을 구현하기 위해 모듈을 요청하는 방법을 배우겠습니다. Elixir에서 이 기능은 비헤이비어라는 이름으로 불립니다.
+
+{% include toc.html %}
+
+## 용도
+
+가끔 공개 API를 공유하는 모듈을 만들어야 할 때가 있습니다. Elixir에서 이런 경우의 해결책은 비헤이비어입니다. 비헤이비어는 두 주요 역할을 수행합니다.
```

Comment:
> ... 만들어야 할 때가 있는데, Elixir에서는 비헤이비어를 사용해서 이런 모듈을 만들 수 있습니다.
> 비헤이비어로 할 수 있는 일은 크게 두 가지가 있습니다.

I'd like to avoid using this(이런 경우) and describe what _this_ means. and little Korean sugars can also make lessons sweet.